src/plugins/intel_gna/src/gna_graph_compiler.cpp

```diff
@@ -1527,7 +1525,7 @@ void GNAGraphCompiler::GemmPrimitive(InferenceEngine::CNNLayerPtr layer) {
     const auto out_dims = outputs->getDims();
     const auto out_dims_size = ngraph::shape_size(out_dims);
     uint32_t num_rows_out = InferenceEngine::GetDimFromBack(out_dims, 1);
-    uint32_t num_padding = ALIGN(num_rows_in, noOfInputsDivisor) - num_rows_in;
+    uint32_t num_padding = ALIGN(num_rows_in, kNoOfInputsDivisor) - num_rows_in;
```

Comment:
it is inconsistent - please use one convention for whole PR or at least for the same file.
In one case There is used `Limitations::` in other not.

![image](https://github.com/openvinotoolkit/openvino/assets/106798706/0c4faf80-e44b-4144-9ab5-0295d923161c)

![image](https://github.com/openvinotoolkit/openvino/assets/106798706/4e031ecf-d2c3-430d-938c-cc13640d3de9)

Reply:
ok